### PR TITLE
Update table-layout

### PR DESCRIPTION
**Pull request by RomanAchilov**
Issue: EPMDJ-4947 EPMDJ-4840 EPMDJ-4838

### CODE REVIEW

File: src/components/table/default-header-cell.module.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import '../../style-guide/index';
+
+.default-header-cell {
+  display: flex;
+  gap: 4px;
```
Comment (RomanAchilov) [inline src/components/table/default-header-cell.module.scss:5]: need to discuss
image

File: src/components/table/table.module.scss [hunk 2]
```diff
@@ -1,30 +1,43 @@
 @import '../../style-guide/index';
 
 .table {
-  border-collapse: collapse;
   width: 100%;
-  font-size: 14px;
-  color: map-get($monochrome, default);
-  white-space: nowrap;
```
Comment (RomanAchilov) [inline src/components/table/table.module.scss:8]: need to discuss

File: src/components/table/table__row.tsx [hunk 3]
```diff
@@ -37,7 +37,7 @@ export const TableRow = BEM(styles).row(
         {columns.map((column) => {
           const Cell = column.Cell || DefaultCell;
           return (
-            <td key={column.name} colSpan={column.colSpan}>
+            <td key={column.name} colSpan={column.colSpan} style={{ width: column.width }} align={column.align}>
```
Comment (RomanAchilov) [inline src/components/table/table__row.tsx:40]: TS dont support `<td width={width} />`

Comment (RomanAchilov) [reply src/components/table/table__row.tsx:40]: fix align

**Review by pavyarov — APPROVED** [review]

**Pull request merged**